packages/core/src/NewAmplify.ts

```diff
@@ -0,0 +1,58 @@
+import {
+	AmplifyUserSession,
+	FrontendConfig,
+	GetUserSessionOptions,
+	ResourceConfig,
+} from './types';
+import { Hub } from './Hub';
+import { Observable } from 'rxjs';
+
+let _resourcesConfig: ResourceConfig | undefined;
+const _frontendConfig: FrontendConfig = { Auth: {} };
+
+export const Amplify = {
+	configure(resources: ResourceConfig, frontendConfig?: FrontendConfig): void {
+		_resourcesConfig = resources;
+		if (frontendConfig) {
+			if (frontendConfig?.Auth === null) {
+				_frontendConfig.Auth = {};
+			} else if (frontendConfig?.Auth) {
+				_frontendConfig.Auth = frontendConfig.Auth;
+			}
+		}
```

Comment:
I wonder if it would be cleaner & more succinct to de-structure `frontendConfig`. Either way I think we can probably remove the outer condition (frontendConfig)

```suggestion
		if (frontendConfig?.Auth === null) {
			_frontendConfig.Auth = {};
		} else if (frontendConfig?.Auth) {
			_frontendConfig.Auth = frontendConfig.Auth;
		}
```

Reply:
I like your approach